control_plane/src/endpoint.rs

```diff
@@ -817,6 +818,7 @@ impl Endpoint {
                 self.http_address.ip(),
                 self.http_address.port()
             ))
+            .header(CONTENT_TYPE.as_str(), "application/json")
```

Comment:
Can you reference the issue? https://github.com/neondatabase/neon/issues/7629

Does it make sense to put the same in all places where we return a json?

Reply:
No, that issue is different, and will be fixed with my Axum work. This commit is in regard to `neon_local` making a request to the management API, while the linked issue is in regard to the management API.